projects/imgix-angular/package.json

```diff
@@ -39,8 +39,8 @@
     "urijs": "1.19.8"
   },
   "peerDependencies": {
-    "@angular/common": "8.x || 9.x || 10.x || 11.x || 12.x",
-    "@angular/core": "8.x || 9.x || 10.x || 11.x || 12.x",
-    "@angular/platform-browser": "8.x || 9.x || 10.x || 11.x || 12.x"
+    "@angular/common": "8.x || 9.x || 10.x || 11.x || 12.x || 13.x",
+    "@angular/core": "8.x || 9.x || 10.x || 11.x || 12.x || 13.x",
+    "@angular/platform-browser": "8.x || 9.x || 10.x || 11.x || 12.x || 13.x"
```

Comment:
Out of curiosity, do we know if these changes allow us to support angular 14 as well?

Reply:
Resolved offline.